site/docs/v1/tech/apis/scim/SCIMServiceProvider.adoc

```diff
@@ -20,18 +19,32 @@ This API is used to retrieve information about the configuration of the link:htt
 
 === Request
 [.api-authentication]
-link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[type=fas]] Retrieve All Resource Types
+link:/docs/v1/tech/apis/authentication#client-credentials[icon:cogs[type=fas]] Retrieve All Resource Types
 [.endpoint]
 .URI
 --
 [method]#GET# [uri]#/api/scim/resource/v2/ResourceTypes#
 --
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#client-credentials[icon:cogs[type=fas]] Retrieve a Resource Type by Id
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/scim/resource/v2/ResourceTypes/`\{resourceTypeId\}`#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#resourceTypeId# [type]#[String]# [optional]#Optional#::
+The unique Resource Type Id, such as `User`.
```

Comment:
Should we document what this does if we omit this parameter? And does this parameter have any constraints, or can it be any value you want? If there are specific values we expect, we should document them.